### PR TITLE
142 compas core update xsd file

### PR DESCRIPTION
**Pull request by SaintierFr**
#142 Update of SCL_CoMPAS.xsd

### CODE REVIEW

File: scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd [hunk 1]
```diff
@@ -13,13 +13,14 @@ SPDX-License-Identifier: Apache-2.0
...
             Version 1.1 Release 2022/01/21
+            Version 2.1 Release 2022/06/08
         </xs:documentation>
     </xs:annotation>
 
     <!--
     SCL Name element.
     -->
-    <xs:simpleType name="tCompasSclName">
+    <xs:simpleType name="tSclName">
```
Comment (dlabordus) [inline scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd:23]: Why is this name Changed? This is now back again to the original name.

File: scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd [hunk 2]
```diff
@@ -29,7 +30,7 @@ SPDX-License-Identifier: Apache-2.0
     <!--
     SCL File Type element.
     -->
-    <xs:simpleType name="tCompasSclFileType">
+    <xs:simpleType name="tSclFileType">
```
Comment (dlabordus) [inline scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd:33]: Same here, why is this name Changed? This is now back again to the original name.

**Review by dlabordus — CHANGES_REQUESTED** [review]

File: scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd (hunk 1 above)
Comment (SaintierFr) [reply scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd:23]: @jeanetiennelemaire @DamienPernet I let you answer

File: scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd (hunk 2 above)
Comment (DamienPERNET) [reply scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd:33]: We have fix this issue and we have normalized all name type like this "tCompas......
[SCL_CoMPAS_20220615.zip](https://github.com/com-pas/compas-core/files/8907311/SCL_CoMPAS_20220615.zip)
"

File: scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd (hunk 1 above)
Comment (SaintierFr) [reply scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd]: Mistake have been corrected in the new version of the xsd file

File: scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd (hunk 2 above)
Comment (SaintierFr) [reply scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd]: Mistake have been corrected in the new version of the xsd file

File: scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd [hunk 3]
```diff
@@ -13,6 +13,7 @@ SPDX-License-Identifier: Apache-2.0
         <xs:documentation xml:lang="en">
             Version 1.0 Release 2021/06/16
             Version 1.1 Release 2022/01/21
+            Version 2.1 Release 2022/06/08
```
Comment (dlabordus) [inline scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd:16]: Maybe change this to Version 1.2, because in the targetNamespace is also still v1.

Comment (SaintierFr) [reply scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd]: Ok, modification done

**Review by dlabordus — APPROVED** [review]

**Pull request merged**